prerendering.bs

```diff
@@ -333,140 +331,123 @@ Every {{Document}} has a <dfn for="Document">post-prerendering activation steps
 
-          - If |historyHandling| is "`default`", clear the forward session history of |predecessorTraversable|, then append |successorEntry| to |predecessorTraversable|'s session history list and make it the active session history entry.
+      1. Let |successorEntry| be |successorTraversable|'s [=traversable navigable/session history entries=][0].
 
-          (Note that as a side effect, this makes |successorBC| the active browsing context of |predecessorTraversable|).
+      1. [=list/Remove=] |successorEntry| from |successorTraversable|'s [=traversable navigable/session history entries=].
 
-       - |successorTraversable| will now be empty and can be unobservably destroyed.
+        <p class="note">At this point, |successorTraversable| is empty and can be unobservably destroyed. (As long as the implementation takes care to keep |successorEntry|, including its [=session history entry/document=] and that {{Document}}'s [=Document/browsing context=], alive for the next step.)
 
-     - This setup means that traversing across the associated history entries will cause a browsing context group switch, similar to what currently happens for \`<a http-header>`Cross-Origin-Opener-Policy`</a>\`. Importantly, this will sever any opener relationships, in the same way as COOP. Such traversal includes both the actual activation process above, where we make |successorEntry| active and thus switch from |predecessorBC| to |successorBC|, but also e.g. the user pressing the back button (or the page using `history.back()`) after activation, which will switch from |successorBC| to |predecessorBC|.
+      1. [=Finalize a cross-document navigation=] given |predecessorTraversable|, |historyHandling|, and |successorEntry|.
 
-  1. [=In parallel=]:
+        <p class="note">Because we have moved the entire [=session history entry=], including the contained [=session history entry/document=]'s [=Document/browsing context=], this means traversing across the associated history entries will cause a browsing context group switch, similar to what happens with the [:Cross-Origin-Opener-Policy:] header. Importantly, this will sever any opener relationships, in the same way as COOP. Such traversal includes both the actual activation process here, where we make |successorEntry| active, but also e.g. the user pressing the back button (or the page using `history.back()`) after activation, which will switch in the other direction.
 
-    1. Update the user agent's user interface to replace |predecessorBC| with |successorBC|, e.g., by updating the tab/window contents and the browser chrome.
+      1. Update the user agent's user interface to reflect this change, e.g., by updating the tab/window contents and the browser chrome.
 
-    1. [=prerendering browsing context/Finalize activation=] for |successorBC| given |predecessorBC|'s [=active document=]'s [=Document/origin=] and |unsafe time|.
```

Comment:
I notice this timestamp is no longer plumbed through; is this basically because the getting of the timestamp and its use are so close together in time as for the difference to be inconsequential?

Reply:
I think the timestamp may have been previously separate because, before https://github.com/whatwg/html/pull/7720 , HTML had some special handling for unload timing? But yes, upon reevaluating this, it seems very unnecessary to grab the timestamp early. Also, I looked at the Chromium code, and the timestamp is computed during the task posted to the renderer process, at the same time we set `document.prerendering` and such.